desktop/libs/indexer/src/indexer/indexers/flink_sql.py

```diff
@@ -47,6 +46,7 @@ def create_table_from_kafka(self, source, destination, start_time=-1, dry_run=Fa
 
     source_type = source['sourceType']
 
+    from desktop.api_public import _get_interpreter_from_dialect   # due to a circular import
```

Comment:
@romainr  is it good now?

Reply:
(and we will see later if we can o better)